README.md

```diff
@@ -442,7 +450,7 @@ $ ecspresso --config ecspresso.yml diff --unified
 -  "platformVersion": "1.3.0"
 +  "platformVersion": "LATEST"
  }
- 
```

Comment:
@koluku revert this change (white space is deleted) please.

Reply:
oh, sorry. I revert it.